.github/workflows/periodic-merge.yml

```diff
@@ -0,0 +1,49 @@
+name: "Periodic Merges"
```

Comment:
Could you add a short comment to the top of this file summarizing what it does and why we need it?

Reply:
Sure, done!